pkg/scheduler/framework/framework.go

```diff
@@ -128,9 +135,13 @@ type frameworkOptions struct {
 	// unselected clusters added to the policy snapshot status.
 	maxUnselectedClusterDecisionCount int
 
-	// checker is the cluster eligibility checker the scheduler framework will use to check
+	// clusterEligibilityChecker is the checker the scheduler framework will use to check
 	// if a cluster is eligibile for resource placement.
 	clusterEligibilityChecker *clustereligibilitychecker.ClusterEligibilityChecker
+
+	// taintTolerationChecker is the checker the scheduler framework will use to check
+	// if taints on a member cluster can be tolerated by tolerations on cluster resource placement.
+	taintTolerationChecker *tainttolerationchecker.TaintTolerationChecker
```

Comment:
This checker will be used for PickFixed placement type similar to clusterEligibilityChecker where we don't run all the filter plugins

Reply:
Discussed offline taints and tolerations plugin will not be run for pickFixed since it's similar to specifying nodeName in podSpec